starboard/nplb/BUILD.gn

```diff
@@ -54,225 +55,238 @@ test("nplb") {
-    "posix_compliance/posix_time_clock_test.cc",
-    "posix_compliance/posix_time_gettimeofday_test.cc",
-    "posix_compliance/posix_time_test.cc",
-    "posix_compliance/posix_time_time_test.cc",
-    "posix_compliance/posix_uname_test.cc",
-    "posix_compliance/posix_unistd_gethostname_test.cc",
-    "posix_compliance/posix_unistd_sysconf_test.cc",
-    "posix_compliance/posix_usleep_test.cc",
-    "random_helpers.cc",
-    "semaphore_test.cc",
-    "storage_close_record_test.cc",
-    "storage_get_record_size_test.cc",
-    "storage_read_record_test.cc",
-    "storage_write_record_test.cc",
-    "system_clear_last_error_test.cc",
-    "system_get_error_string_test.cc",
-    "system_get_extension_test.cc",
-    "system_get_last_error_test.cc",
-    "system_get_locale_id_test.cc",
-    "system_get_number_of_processors_test.cc",
-    "system_get_path_test.cc",
-    "system_get_property_test.cc",
-    "system_get_random_data_test.cc",
-    "system_get_random_uint64_test.cc",
-    "system_get_stack_test.cc",
-    "system_get_total_cpu_memory_test.cc",
-    "system_get_total_gpu_memory_test.cc",
-    "system_get_used_cpu_memory_test.cc",
-    "system_get_used_gpu_memory_test.cc",
-    "system_has_capability_test.cc",
-    "system_hide_splash_screen_test.cc",
-    "system_is_debugger_attached_test.cc",
-    "system_network_status_test.cc",
-    "system_sign_with_certification_secret_key_test.cc",
-    "system_symbolize_test.cc",
-    "thread_get_id_test.cc",
-    "thread_priority_test.cc",
-    "thread_sampler_test.cc",
-    "thread_test.cc",
-    "time_zone_get_name_test.cc",
-    "ui_navigation_test.cc",
-    "undefined_behavior_test.cc",
-    "url_player_create_test.cc",
   ]
 
-  if (!is_androidtv) {
+  sources = [ "//starboard/common/test_main.cc" ]
+
+  # All platforms use the Starboard Media functionality.
+  sources += media_sources
+
+  if (is_androidtv) {
+    # TODO(mcasas): These two are just placeholders for SbWindowCreate/Destroy
+    # methods which are needed to build media_player_files on is_androidtv
+    # platforms. Use the starboard/android ones instead.
     sources += [
-      "//starboard/nplb/sabi/alignment_test.cc",
-      "//starboard/nplb/sabi/endianness_test.cc",
-      "//starboard/nplb/sabi/signedness_and_size_of_enum_test.cc",
-      "//starboard/nplb/sabi/signedness_of_char_test.cc",
-      "//starboard/nplb/sabi/size_test.cc",
-      "//starboard/nplb/sabi/struct_alignment_test.cc",
-      "//starboard/testing/fake_graphics_context_provider.cc",
-      "//starboard/testing/fake_graphics_context_provider.h",
-      "maximum_player_configuration_explorer_test.cc",
-      "media_set_audio_write_duration_test.cc",
-      "multiple_player_test.cc",
-      "player_create_test.cc",
-      "player_get_audio_configuration_test.cc",
-      "player_get_media_time_test.cc",
-      "player_test_fixture.cc",
-      "player_test_fixture.h",
-      "player_write_sample_test.cc",
+      "//starboard/shared/stub/window_create.cc",
```

Comment:
Just one late drive-by comment: I'm not sure if this works as expected. After trying to do the same for tvOS, I ended up hitting [this DCHECK](https://github.com/youtube/cobalt/blob/6673561090776ec6de7378e40a53ef2570044a1a/starboard/testing/fake_graphics_context_provider.cc#L191) because the stub SbWindowCreate implementation just returns an invalid handle.